lookout/style/typos/cmdline_tools.py

```diff
@@ -139,7 +139,7 @@ def add_parser(name, help):
     # One command to rule them all
     ########################################
     train_from_scratch_parser = add_parser(
-        "train-from-scratch", "Create and train TyposCorrector  model on the given data.")
+        "train-from-scratch", "Create and train TyposCorrector model on the given data.")
```

Comment:
At this point I have got the idea of a very very simple Lookout analyzer to find these goddamn double whitespaces everywhere and bust them. I have hit this problem again and again since many years ago. @marnovo @campoy @smola